docs/reference/glossary.asciidoc

```diff
@@ -61,6 +61,16 @@
   `object`. The mapping also allows you to define (amongst other things)
   how the value for a field should be analyzed.
 
+[[glossary-filter]] filter ::
+
+  A filter is a query. It is a kind of query which does not give a score, so it is 
+  known as a "non-scoring" query.It is only concerned about answering the question - 
```

Comment:
nit: whitespace between "query.It". Although the difference won't show up in the docs I think, it improves readability here.

Reply:
Thanks, @cbuescher I will work on your review. As a beginner to Elasticsearch myself, I appreciate the importance of getting the glossary right. I'll be happy to get it right.